server/comms/server.go

```diff
@@ -450,20 +450,30 @@ func (s *Server) Run(ctx context.Context) {
 		rr.Get("/config", routeHandler(msgjson.ConfigRoute))
 		rr.Get("/spots", routeHandler(msgjson.SpotsRoute))
 		rr.Get("/logo", func(w http.ResponseWriter, r *http.Request) {
+			writeErr := func(code int, msg string) {
+				w.WriteHeader(code)
+				w.Write([]byte(msg))
+			}
+
 			handler := httpRoutes[msgjson.LogoRoute]
 			if handler == nil {
-				writeServerError(w, map[string]string{"error": "no handler"})
+				log.Critical("No handler for %s", msgjson.LogoRoute)
+				writeErr(500, "Something unexpected happened")
```

Comment:
`http.StatusInternalServerError`

Could also use `http.StatusText(http.StatusInternalServerError)` for the text.